tests/integration/cheatcodes/expect_call/expect_call_test.cairo

```diff
@@ -0,0 +1,87 @@
+%lang starknet
+
+@contract_interface
+namespace MainContract {
+  func get_id() -> (res: felt) {
+  }
+  func increase_balance(amount_1: felt, amount_2: felt, amount_3: felt) -> () {
+  }
+  func get_balance() -> (res: felt) {
+  }
+}
+
+@external
+func __setup__() {
+  %{ context.ctr_addr_a = deploy_contract("./tests/integration/cheatcodes/expect_call/basic_contract.cairo", [123, 100]).contract_address %}
+  %{
+    declared = declare("./tests/integration/cheatcodes/expect_call/basic_contract.cairo")
+    prepared = prepare(declared, [124, 120])
+    context.ctr_addr_b = deploy(prepared).contract_address
+  %}
+
+  return ();
+}
+
+@external
+func test_expect_call_success{syscall_ptr: felt*, range_check_ptr}() {
+  tempvar ctr_addr_a;
+  %{ ids.ctr_addr_a = context.ctr_addr_a %}
+  tempvar ctr_addr_b;
+  %{ ids.ctr_addr_b = context.ctr_addr_b %}
+  %{
+    expect_call(ids.ctr_addr_a, [5, 6, 7])
+    expect_call(ids.ctr_addr_b, [1, 2, 3])
+  %}
+
+  MainContract.increase_balance(contract_address=ctr_addr_a, amount_1=5, amount_2=6, amount_3=7);
+  MainContract.increase_balance(contract_address=ctr_addr_b, amount_1=1, amount_2=2, amount_3=3);
+
+  return ();
+}
+
+@external
+func test_expect_call_partial_success{syscall_ptr: felt*, range_check_ptr}() {
```

Comment:
[Functionality] There should be no partial success. If there's a mismatch somewhere, the test should fail.

Reply:
ok

Reply:
[remove redundant, contract_address, remove partial match](https://github.com/software-mansion/protostar/pull/1060/commits/5fc611bd5ecd9f920835f79dd3d0e4a951e374fe)